packages/x-components/src/views/home/Home.vue

```diff
@@ -260,7 +268,7 @@
                         :max="controls.slicedFilters.max"
                         :data-test="`${facet.label}-sliced-filters`"
                       >
-                        <SelectedFilters #default="{ selectedFilters }" :facetsIds="[facet.id]">
+                        <SelectedFilters v-slot="{ selectedFilters }" :facetsIds="[facet.id]">
```

Comment:
I guess it was auto-fixed by eslint, we should keep with `#default`.
```suggestion
                        <SelectedFilters #default="{ selectedFilters }" :facetsIds="[facet.id]">
```